### PR TITLE
NearBy 

### PR DESCRIPTION
**Pull request by Dheeru12503**
Check it

### CODE REVIEW

**Comment by Dheeru12503** [comment]
All done in nearBy section and check it , only One problem in TimeBox in nearBySection.

File: src/app/components/NearBy/NearBy.tsx [hunk 1]
```diff
@@ -38,46 +38,29 @@ export default function NearSection() {
...
               </div>
             ))}
           </div>
 
-          <div className={`${styles.rightSection} ${styles.rightSectionLg}`}>
-            <div className={`${styles.mapWrapper} ${styles.mapWrapperLg}`}>
+          <div className={styles.rightSection}>
+            <div>
               <iframe
                 src={locationUrl}
```
Comment (mohdnoorain) [inline src/app/components/NearBy/NearBy.tsx]: @Dheeru12503 
check console this is generating error.
just fix this and all good

File: src/app/components/NearBy/NearBy.module.css [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  padding-bottom: 100px;
+  background-color: #f7f7f7;
+ 
+}
+
+.container {
+  margin: 0 auto;
+}
+
+@media (min-width: 200px) {
```
Comment (mohdnoorain) [inline src/app/components/NearBy/NearBy.module.css]: @ no need for this 200px media query

File: src/app/components/Amenities/Amenities.tsx [hunk 3]
```diff
@@ -33,8 +35,8 @@ export default function Amenities() {
           <ul className={styles.flexWrapper}>
             {Amenities.map((item) => (
               <li className={styles.amenityItem} key={item}>
```
Comment (mohdnoorain) [inline src/app/components/Amenities/Amenities.tsx]: @Dheeru12503 
you can make this a component if it's very small create this in this the same file.

**Review by mohdnoorain — CHANGES_REQUESTED** [review]

File: src/app/components/NearBy/NearBy.module.css (hunk 2 above)
Comment (Dheeru12503) [reply src/app/components/NearBy/NearBy.module.css]: done

File: src/app/components/NearBy/NearBy.tsx (hunk 1 above)
Comment (Dheeru12503) [reply src/app/components/NearBy/NearBy.tsx]: this two line is not showing in my code.

Comment (Dheeru12503) [reply src/app/components/NearBy/NearBy.tsx]: issues is coming in 
 <iframe
                src={locationUrl || "about:blank"}  // it error problem 
                width="100%"
                allowFullScreen
                className={styles.mapWrapper}
                loading="lazy"
              ></iframe>
              
              but is fix by adding this "about:blank"

File: src/app/components/Amenities/Amenities.tsx (hunk 3 above)
Comment (Dheeru12503) [reply src/app/components/Amenities/Amenities.tsx]: done

Comment (mohdnoorain) [reply src/app/components/Amenities/Amenities.tsx]: @Dheeru12503 
see again "create this in this the same file."

Comment (Dheeru12503) [reply src/app/components/Amenities/Amenities.tsx]: Done and Check it

**Review by mohdnoorain — APPROVED** [review]

**Pull request merged**